src/css/tailwind.css

```diff
@@ -1,32 +1,35 @@
 @tailwind base;
 
-a {
-  @apply text-primary;
-}
-a:hover {
-  @apply text-primary-darker;
-}
+@layer base {
```

Comment:
See mention of `@layer` in Tailwind documentation: https://tailwindcss.com/docs/adding-base-styles